gui/wxpython/vdigit/toolbars.py

```diff
@@ -247,97 +247,139 @@ def _toolbarData(self):
+                (
+                    ("addVertex", _("Add new vertex to line or boundary")),
+                    self.icons["addVertex"],
+                    self.OnAddVertex,
+                    wx.ITEM_CHECK,
+                ),
             )
         if not self.tools or "removeVertex" in self.tools:
             data.append(
                 (
-                    "removeVertex",
+                    ("removeVertex", _("Remove selected vertex")),
                     self.icons["removeVertex"],
                     self.OnRemoveVertex,
                     wx.ITEM_CHECK,
-                )
+                ),
             )
         if not self.tools or "editLine" in self.tools:
             data.append(
-                ("editLine", self.icons["editLine"], self.OnEditLine, wx.ITEM_CHECK)
+                (
+                    ("editLine", _("Edit selected line/boundary")),
+                    self.icons["editLine"],
+                    self.OnEditLine,
+                    wx.ITEM_CHECK,
+                ),
             )
         if not self.tools or "moveLine" in self.tools:
             data.append(
-                ("moveLine", self.icons["moveLine"], self.OnMoveLine, wx.ITEM_CHECK)
+                (
+                    ("moveLine", _("Move selected feature(s)")),
+                    self.icons["moveLine"],
+                    self.OnMoveLine,
+                    wx.ITEM_CHECK,
+                ),
             )
         if not self.tools or "displayCats" in self.tools:
             data.append(
                 (
-                    "displayCats",
+                    ("displayCats", _("Display/update categories")),
                     self.icons["displayCats"],
                     self.OnDisplayCats,
                     wx.ITEM_CHECK,
-                )
+                ),
             )
         if not self.tools or "displayAttr" in self.tools:
             data.append(
                 (
-                    "displayAttr",
+                    ("displayAttr", _("Display/update attributes")),
                     self.icons["displayAttr"],
                     self.OnDisplayAttr,
                     wx.ITEM_CHECK,
-                )
+                ),
             )
         if not self.tools or "additionalSelf.Tools" in self.tools:
             data.append(
                 (
-                    "additionalTools",
+                    ("additionalTools", _("Additional tools")),
                     self.icons["additionalTools"],
                     self.OnAdditionalToolMenu,
                     wx.ITEM_CHECK,
-                )
+                ),
             )
         if not self.tools or "undo" in self.tools or "redo" in self.tools:
             data.append((None,))
         if not self.tools or "undo" in self.tools:
-            data.append(("undo", self.icons["undo"], self.OnUndo))
+            data.append(
+                (
+                    ("undo", _("Undo previous changes")),
```

Comment:
Shouldn't this be singular? i.e., it will undo one by one, no?

Reply:
Yes I agree. I'm copy this label text from another tool label.